lib/QrCamera.dart

```diff
@@ -1,183 +0,0 @@
-import 'dart:async';
```

Comment:
You should have used `git mv` to rename files. That way the change history should be kept.